pages/clusters/manage_clusters.md

```diff
@@ -63,6 +63,22 @@ To add a maintainer to a cluster:
 1. Select a user or team.
 1. Click _Add Maintainer_.
 
+### Restrict access for a cluster token by IP address
+
+Each cluster token can be locked down so that only agents with an allowed IP address can use them to register.
+
+You can set the _Allowed IP Addresses_ when creating a token, or you can modify existing tokens:
+
+1. Navigate to the cluster's _Agent Tokens_.
+1. Select the token to which you wish to restrict access.
+1. Select _Edit_.
+1. Update the _Allowed IP Addresses_ setting, using [CIDR notation](https://en.wikipedia.org/wiki/Classless_Inter-Domain_Routing).
```

Comment:
Do we want to clarify how to specify multiple CIDR ranges? I assume it's comma separated? Maybe the UI makes it clear, I haven't peeked just now.

Reply:
Yeah, the UI shows that:
![CleanShot 2023-06-27 at 12 47 22](https://github.com/buildkite/docs/assets/11305534/ba834db6-7e03-4dba-bcb7-e3db4c4d6845)

but I'll add `space-separated` here

Reply:
Not really a docs question, but does the implementation also allow splitting by comma? People will totally comma separate them without reading the UI text :)

I guess I can check…

<img width="836" alt="image" src="https://github.com/buildkite/docs/assets/15759/5cd4d42a-0ee5-451d-afbd-f8e87e624e7c">

Nope it doesn't allow commas, but at least we're consistent across all `Buildkite::IPAddrList` usage :)
And the UI makes it reasonably clear what's wrong 👍🏼

Reply:
Yeah, I opted for consistency there. Lack of comma-separation was a little weird, but it's easily enough to add across the board if we get feedback about it